ACadSharp/IO/Templates/CadArcTemplate.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using ACadSharp.Entities;
+using ACadSharp.IO.DXF;
+using ACadSharp.Tables;
+
+namespace ACadSharp.IO.Templates
+{
+	internal class CadArcTemplate : CadEntityTemplate
+	{
+
+		public CadArcTemplate(Entity entity) : base(entity) { }
+
+
+        public override void Build(CadDocumentBuilder builder)
+		{
+			base.Build(builder);
+
+            if (builder is DxfDocumentBuilder && this.CadObject is Arc arc)
+            {
+                arc.StartAngle *= MathUtils.DegToRad;
+                arc.EndAngle *= MathUtils.DegToRad;
```

Comment:
Instead of checking if the builder is a `DxfDocumentBuilder` I would prefer to override the method:
```CSharp
public virtual bool CheckDxfCode(int dxfcode, object value)
```
This method bypassed the mapper when the DxfReader assigns the values, this way we'll keep the assignation in the dxf scope.